features/_trustedboot/file.include/etc/update-motd.d/25-secureboot

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -eufo pipefail
+
+secureboot_state=0
+efivars_path=/sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c
```

Comment:
Could that path change?

Reply:
nope, that's how efivars work, they are store in firmware under 2 keys, vedor UUID + name, and the linux kernel maps this to `$name-$uuid` in the efivars fs

only if the kernel ever decides to change this, but "don't break userspace", so not likely
the secureboot UUID is part of EFI spec, so this won't change unless some new EFI version with new incompatible spec